x-pack/test/security_solution_endpoint_api_int/apis/endpoint_artifacts/host_isolation_exceptions.ts

```diff
@@ -64,57 +49,115 @@ export default function ({ getService }: FtrProviderContext) {
         }
       };
     };
+    type UnknownBodyGetter = () => unknown;
+    type PutPostBodyGetter = (
+      overrides?: Partial<ExceptionListItemSchema>
+    ) => Pick<
+      ExceptionListItemSchema,
+      'item_id' | 'namespace_type' | 'os_types' | 'tags' | 'entries'
+    >;
+    type HostIsolationExceptionApiCallsInterface<BodyGetter = UnknownBodyGetter> = Array<{
+      method: keyof Pick<typeof supertest, 'post' | 'put' | 'get' | 'delete' | 'patch'>;
```

Comment:
Don't have to change but you could use `RouteMethod` here instead that is imported as

`import type { RouteMethod } from '@kbn/core-http-server';`